src/Footer.jsx

```diff
@@ -0,0 +1,239 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import locales from 'mwp-config/locales';
+
+import Bounds from './layout/Bounds';
+import Chunk from './layout/Chunk';
+import Flex from './layout/Flex';
+import FlexItem from './layout/FlexItem';
+import InlineBlockList from './layout/InlineBlockList';
+import GridList from './layout/GridList';
+import Icon from './media/Icon';
+import Section from './layout/Section';
+import SelectInput from './forms/SelectInput';
+import Stripe from './layout/Stripe';
+import withMatchMedia from './utils/components/withMatchMedia';
+
+import { getSocialLinks } from './utils/getSocialLinks';
+
+export const FooterCategory = (props) => (
+	<Chunk>
+		<h4 className="text--bold margin--bottom">{props.header}</h4>
+		<GridList
+			columns={{
+				default: 2,
+				medium: 1,
+			}}
+			className="text--secondary text--small"
+			itemClassNames="footer-item"
+			items={props.items}
+		/>
+	</Chunk>
+);
+
+
+export const SocialIconsList = (language) => {
+	const socialLinks = getSocialLinks(language);
+	const socialIcons = [
+		<a href={socialLinks.facebook}>
+			<Icon shape="external-facebookboxed" size="s" />
+		</a>,
+		<a href={socialLinks.twitter}>
+			<Icon className="margin--left" shape="external-twitter" size="s" />
+		</a>,
+		<a href={socialLinks.youtube}>
+			<Icon className="margin--left" shape="external-youtube" size="s" />
+		</a>,
+		<a href={socialLinks.instagram}>
+			<Icon className="margin--left" shape="external-instagram" size="s" />
+		</a>,
+		<a href={socialLinks.medium}>
+			<Icon className="margin--left" shape="external-medium" size="s" />
+		</a>,
+	];
+
+	return <InlineBlockList items={socialIcons} />;
+};
+
+// this should probably be folded into mwp-cli, where lang config is kept
+const languageMap = {
+	'en-US': 'English',
+	'en-AU': 'English (Australia)',
+	'de-DE': 'Deutsch',
+	es: 'Español',
+	'es-ES': 'Español (España)',
+	'fr-FR': 'Français',
+	'it-IT': 'Italiano',
+	'nl-NL': 'Nederlands',
+	'pl-PL': 'Polski',
+	'pt-BR': 'Português',
+	'tr-TR': 'Türkçe',
+	'th-TH': 'ไทย',
+	'ja-JP': '日本語',
+	'ko-KR': '한국어',
+	'ru-RU': 'Русский',
+};
+
+/*
+ * Turn list of languages into an array of options for select dropdown
```

Comment:
very minor style nitpick; the comment here would be unnecessary if the map was happening closer to where it's being used:

```
options={locales
     .map(lang => ({
          label: languageMap[lang],
	      value: lang,
     })
)}
```

Reply:
This makes the test `it('should render the correct language options')` more difficult to test for. I still have 100% coverage after removing that test, but I'm open to alternatives for testing if you think it's important to pass data to `options` this way **and** test to make sure the options match